packages/template-retail-react-app/app/components/item-variant/item-attributes.jsx

```diff
@@ -38,25 +40,68 @@ const ItemAttributes = ({includeQuantity, currency, ...props}) => {
         }
     )
     const promos = res?.data || []
-    // Create a mapping of variation values to their associated attributes. This allows us
-    // the render the readable names/labels rather than variation value IDs.
-    const variationValues = Object.keys(variant.variationValues || []).map((key) => {
-        const value = variant.variationValues[key]
-        const attr = variant.variationAttributes?.find((attr) => attr.id === key)
-        return {
-            id: key,
-            name: attr?.name || key,
-            value: attr.values.find((val) => val.value === value)?.name || value
+    const variationValues = getDisplayVariationValues(
```

Comment:
I think implementation is different as the return type from `getDisplayVariationValues` and the return type from the previous implementation look different

`getDisplayVariantValues` returns an object that looks like `{ "Colour": "royal" }`

while the original seems to returns an array that looks like: `[{ id: 'id', name: 'name', value: 'value'}]`

Is this breaking?